rfc/2/index.md

```diff
@@ -130,7 +130,7 @@ In the future, the OME-Zarr community MAY decide to restrict the allowed feature
 
 While the adoption of Zarr v3 does not strictly require changes to the OME-Zarr metadata, this proposal contains changes to align with community conventions and ease implementation:
 
-- OME-Zarr metadata will be stored under a dedicated key in the Zarr array or group attributes. The key will be a well-known URI of the OME-NGFF specification with a version number, e.g. `https://ngff.openmicroscopy.org/0.6`.
+- OME-Zarr metadata will be stored under a dedicated key in the Zarr array or group attributes. The key will be a well-known URI of the OME-NGFF specification with a version number, e.g. `https://ngff.openmicroscopy.org/0.5`.
```

Comment:
This sentence needs updating to the `ome` key now.
Also I wonder if you could update the PR description too, thanks.

Reply:
This change will be superseded by #250 
```suggestion
- OME-Zarr metadata will be stored under a dedicated key in the Zarr array or group attributes. The key will be a well-known URI of the OME-NGFF specification with a version number, e.g. `https://ngff.openmicroscopy.org/0.6`.
```

Reply:
OK, understood, thanks.